libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/tracing/TraceLogPack.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+package io.element.android.libraries.matrix.api.tracing
+
+enum class TraceLogPack(val key: String) {
+    EVENT_CACHE("event_cache") {
+        override val title: String = "Event Cache"
+    },
+    SEND_QUEUE("send_queue") {
+        override val title: String = "Send Queue"
+    },
+    TIMELINE("timeline") {
+        override val title: String = "Timeline"
+    };
+
+    abstract val title: String
```

Comment:
Why treating diffentely the `key` and the `title`? Is it for (future?) i18n support?

Reply:
Yeah, mostly so if we add a typo by mistake it wouldn't save/load it as a different value.

Reply:
I see. The enum is serialized and stored. That a sufficient reason!